server/lobbyconnection.py

```diff
@@ -121,7 +126,15 @@ async def abort(self, logspam=""):
 
     async def ensure_authenticated(self, cmd):
         if not self._authenticated:
-            if cmd not in ["hello", "ask_session", "create_account", "ping", "pong", "Bottleneck"]:  # Bottleneck is sent by the game during reconnect
+            if cmd not in (
+                "ask_session",
+                "auth",
+                "Bottleneck"  # Bottleneck is sent by the game during reconnect
```

Comment:
Oh wow, here's a nasty bug. What happens when you forget a comma:
```python3
>>> (
...     "ask_session",
...     "auth",
...     "Bottleneck"  # Bottleneck is sent by the game during reconnect
...     "create_account",
...     "hello",
...     "ping",
...     "pong",
... )
('ask_session', 'auth', 'Bottleneckcreate_account', 'hello', 'ping', 'pong')
```

Reply:
can you even create an account through the server anymore? Didn't realize that was a command.

Reply:
No, the command just returns an error message